src/scopes/grants/index.test.js

```diff
@@ -227,17 +227,29 @@ describe('grant filtersToScopes', () => {
     it('filters by', async () => {
       const filters = { 'programType.in': ['EHS'] };
       const scope = filtersToScopes(filters);
-      const found = await Grant.findAll({
-        where: { [Op.and]: [scope.grant, { id: possibleIds }] },
+      const found = await Recipient.findAll({
```

Comment:
The problem with calling Grant.findAll() directly from the test is that it expects the alias to be "Grant". But they way we are using scopes.grant in the overview widget it expects "grants" alias.

I think the solution here is to update the test to match how we are using it.